eclair-node/pom.xml

```diff
@@ -137,6 +137,12 @@
             <version>${akka.http.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why is this new dependency needed?

Reply:
It's required at runtime during tests by akka http, for some reason. If omitted I get this:

```
 Cause: java.lang.NoClassDefFoundError: akka/stream/testkit/TestSubscriber$
  at akka.http.scaladsl.testkit.WSProbe$$anon$1.<init>(WSProbe.scala:107)
  at akka.http.scaladsl.testkit.WSProbe$.apply(WSProbe.scala:106)
  at fr.acinq.eclair.api.ApiServiceSpec.$anonfun$new$58(ApiServiceSpec.scala:469)
  at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
  at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
  at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
  at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
  at org.scalatest.Transformer.apply(Transformer.scala:22)
  at org.scalatest.Transformer.apply(Transformer.scala:20)
  at org.scalatest.funsuite.AnyFunSuiteLike$$anon$1.apply(AnyFunSuiteLike.scala:189)
```